### PR TITLE
small fixes for QA stability when run 100s of times

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

**Comment by ptschip** [comment]
you've got a fomatting issue in init.cpp

**Review by ptschip — APPROVED** [review]

File: src/Makefile.am [hunk 1]
```diff
@@ -34,6 +34,10 @@ endif
 
 EXTRA_LIBRARIES =
 
+AM_CXXFLAGS += -O0 -g
+AM_CPPFLAGS += -O0 -g
+AM_LDFLAGS  += -O0 -g
```
Comment (sickpig) [inline src/Makefile.am]: Those changes are unrelated to the QA python framework.

Comment (sickpig) [reply src/Makefile.am]: Maybe the timeout we are seeing on travis for builds that usually take a lot less time are due to this change.

E.g. in the following build you see how we don't even reach the end of bitcoin-qt compilation whereas for this conf hte compilation time is around 10-15 mins.

https://travis-ci.org/BitcoinUnlimited/BitcoinUnlimited/jobs/593114808

Comment (gandrewstone) [reply src/Makefile.am]: nice catch!  This was added temporarily for easier debugging
